jupyter-extension/src/plugins/mount/mount.tsx

```diff
@@ -614,20 +459,16 @@ export class MountPlugin implements IMountPlugin {
     await this._poller.refresh();
 
     if (this._poller.status.code === 500) {
-      this.setShowFullPageError(true);
+      console.error('poller status 500');
+      // this.setShowFullPageError(true);
     } else {
-      this.setShowConfig(
-        this._poller.config.cluster_status === 'INVALID' ||
-          this._poller.status.code !== 200,
-      );
-
       if (this._poller.status.code === 200) {
         try {
           const res = await requestAPI<CurrentDatumResponse>('datums', 'GET');
           if (res['num_datums'] > 0) {
             this._keepMounted = true;
             this._currentDatumInfo = res;
-            await this.setShowDatum(true);
+            this.setCurrentView(this._datumPanel);
           }
         } catch (e) {
           console.log(e);
```

Comment:
Similar to my previous comment, I wonder if we want to start surfacing these errors to the user rather than just logging them? (I know it's not related strictly to this just saw it while reading)

Reply:
I agree 100%, but I think addressing this is best suited for a subsequent PR.

Reply:
Agreed 👍🏻